verticapy/sql/functions/conditional.py

```diff
@@ -32,16 +32,50 @@ def case_when(*args) -> StringSQL:
     args: SQLExpression
```

Comment:
**For everything below:**

Same as above:

change verticapy.stats by verticapy.sql.functions as vpf
bring the comment of the function usage out the code (create a new block)
change TableSample by vDataFrame (so you have to change your import as well)
Try to follow PEP
Use a new line after the last import